### ISSUE TITLE
Parentheses on figured bass

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/music-encoding#619]
`@enclose` isn't allowed (yet) on `<fb>` and `<f>`. How should these numbers be encoded with parentheses? 
```xml
<fb>
  (<f>5</f>
  <f>3</f>)
</fb>
```
Probably something like this would make sense, but it still doesn't look right.

**Comment by lpugin** [comment]
I agree it would make sense to have it on `<f>`. I am less sure about `<fb>`. 

For now I would probably do:
```xml
<fb>
  <f>(5)</f>
  <f>(3)</f>
</fb>
```
Or, maybe better, use editorial markup, such as `supplied`

**Comment by egorpol** [comment]
Sorry for bumping up the old issue, I have a concrete case for this same problem: a source with parentheses around a complete figured-bass stack, not around each individual figure. This is not just a question of where to type literal parentheses; the desired visual result is a group-level enclosure spanning the complete stack.

Facsimile crop:

<img width="250" height="500" alt="Image" src="https://github.com/user-attachments/assets/0eb4befc-5ee6-42de-9e5f-3aae93c655bd" />

The source case is from:

- Repository: https://github.com/egorpol/DdT_1_vol_11
- File: `11_buxtehude_dietrich_buxtehudes_instrumentalwerke_bsb00023199/bsb00023199_00086_facs_zones.mei`
- Measure: `n="47"`, `xml:id="ggk09au"`
- Measure facsimile zone: `<zone xml:id="zone_1e02283eefa342fbb8" type="measure" ulx="3263" uly="3994" lrx="3631" lry="5332" />`
- Harmony: `xml:id="as8v3t"`, `staff="3"`, `tstamp="1"`
- IIIF facsimile crop: https://api.digitale-sammlungen.de/iiif/image/v2/bsb00023199_00086/4468,6195,368,820/1300,/0/default.jpg

The semantic target is a single parenthesized stack:

```text
(6
 4
 2)
```

This is different from parenthesizing each figure separately:

```xml
<fb>
  <f>(6)</f>
  <f>(4)</f>
  <f>(2)</f>
</fb>
```

I also tested a literal-outer-parentheses in Verovio `6.1.0-682d606`:

```xml
<harm xml:id="as8v3t" staff="3" tstamp="1">
  <fb>
    <f>(6</f>
    <f>4</f>
    <f>2)</f>
  </fb>
</harm>
```

This is better than parenthesizing every `f`, but it still does not enclose the whole stack. It only puts a literal left parenthesis next to the top figure and a literal right parenthesis next to the bottom figure:

<img width="816" height="302" alt="Image" src="https://github.com/user-attachments/assets/e66cfff3-a044-4644-924d-a0263809bca7" />

In that comparison, the columns are:

1. plain `6`, `4`, `2`
2. literal outer parentheses: `<f>(6</f> ... <f>2)</f>`
3. individual parentheses: `<f>(6)</f>`, `<f>(4)</f>`, `<f>(2)</f>`

Would it make sense for MEI to support a group-level enclosure on `fb`, for example via `@enclose`, or is there another preferred encoding pattern for this case?

**Comment by pe-ro** [comment]
Since `<harm>` allows a mixture of text and child elements, the following will validate --
```
<harm xml:id="as8v3t" staff="3" tstamp="1">
  (<fb>
    <f>6</f>
    <f>4</f>
    <f>2</f>
  </fb>)
</harm>
```
However, this will likely still result in the parentheses being displayed before the initial figure and after the last figure, not surrounding the entire `<fb>`.

At present, `<fb>` isn't allowed within `<supplied>`, so the following markup won't validate --
```
<harm xml:id="as8v3t" staff="3" tstamp="1">
  <supplied>
    <fb>
      <f>6</f>
      <f>4</f>
      <f>2</f>
    </fb>
  </supplied>
</harm>
```
@lpugin's suggestion of `<supplied>` (or some other editorial/transcriptional element) is the best approach since it captures the semantics of the parentheses --
```
<measure>
  <supplied>
    <harm xml:id="as8v3t" staff="3" tstamp="1">
      <fb>
        <f>6</f>
        <f>4</f>
        <f>2</f>
      </fb>
    </harm>
  </supplied>
</measure>
```
I think this is the best approach, but I'm not opposed to the idea of adding `@enclose` to `<harm>`, mostly to accommodate our favorite example of a naive encoder, OMR.
